apps/ehr/src/components/patients-merge/queries.ts

```diff
@@ -29,6 +29,7 @@ export const useGetPatientsForMerge = (
       throw new Error('fhir client not defined or patientIds not provided');
     },
     {
+      enabled: oystehr !== undefined && patientIds !== undefined,
```

Comment:
I suggest using either just `oystehr`, `Boolean(oystehr)`, or `!!oystehr` instead of `oystehr !== undefined` for cases like that. We need any truthy value there, not specifically non-undefined.